gnat2goto/driver/tree_walk.adb

```diff
@@ -1763,9 +1732,11 @@ package body Tree_Walk is
          Jump_Irep := Make_Code_Break (Source_Location => Sloc (N));
       end if;
       if Present (Condition (N)) then
-         return R : constant Irep := Do_When (Condition (N)) do
-            Set_Then_Case (R, Jump_Irep);
-         end return;
+         return Make_Code_Ifthenelse
+           (Source_Location => Sloc (N),
+            Cond => Do_Expression (Condition (N)),
+            Then_Case => Jump_Irep,
+            Else_Case => Ireps.Empty);
```

Comment:
`Make_Nil`? Or am I wrong?

Reply:
I'm mirroring the previous code where they were left empty. Not sure if it'd also work with nil (the thing about nil is that it's actually _something_ that is nil, whereas with empty that just means the value isn't set to anything)